mantle/platform/api/aliyun/api.go

```diff
@@ -207,7 +212,6 @@ func (a *API) ImportImage(format, bucket, object, image_size, device, name, desc
 	request := ecs.CreateImportImageRequest()
 	request.SetConnectTimeout(defaultConnectTimeout)
 	request.SetReadTimeout(defaultReadTimeout)
-	request.Scheme = "https"
```

Comment:
Isnt this supposed to be changed to HTTPS?

Reply:
Yes, it should be. But I'm removing the line here altogether, so it shouldn't matter.